.travis.yml

```diff
@@ -93,6 +93,8 @@ install:
     fi
   - virtualenv "$STARFORGE_VENV"
   - . "$STARFORGE_VENV"/bin/activate
+  - pip --version
```

Comment:
By looking at the Travis logs, here `pip` is already at the latest version also for the failing macOS jobs.